asabiris/orchestration/sendslack.py

```diff
@@ -123,3 +118,17 @@ def get_content_type(self, file_extension):
 		"""
 		content_type = mimetypes.guess_type('dummy' + file_extension)[0]
 		return content_type or 'application/octet-stream'
+
+	def _generate_error_message_slack(self, specific_error: str) -> Tuple[str, str]:
+		timestamp = datetime.datetime.now(tz=datetime.timezone.utc).strftime("%Y-%m-%d %H:%M:%S")
+		error_message = (
+			":warning: *Hello!*\n\n"
+			"We encountered an issue while processing your request:\n`{}`\n\n"
```

Comment:
You want to send the specific error to the end user (customer!) only if they can do something about that. If it says there is wrong configuration if the BE service (for example), it is not helpful.